app/controllers/LilaController.scala

```diff
@@ -359,7 +359,7 @@ private[controllers] trait LilaController
             hasFingerprint = hasFingerprint,
             assetVersion = getAssetVersion,
             inquiry = inquiry,
-            nonce = Nonce get isPage)
+            nonce = Nonce.forRequest(ctx.req))
```

Comment:
the point here is that `isPage` was already computed, and I wanted to reuse it.

Reply:
i saw that, but it looked like premature optimization. should i add it back?